foundations/tying_it_all_together/conclusion.md

```diff
@@ -1,43 +1,44 @@
 
 With what you know already you could start putting together some basic websites, but we're here to help you get a lot further than that. We want to bring you to the point where you can confidently build a full-featured and scalable web application, either on your own, or working as a developer in a top-tier tech company.  
 
-You've come a long way up the learning curve but there is so much more cool stuff that you'll learn to do, and now is the time to decide which path you want to take. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects, and becoming a great developer. There is no right answer here on what is the next best path. Learning multiple languages can help you become a well rounded developer, but sticking with one could make you more of an expert in the language itself. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
+You've come a long way but there is so much more cool stuff that you'll learn to do. Now is the time to decide which path you want to take going forward. Either Ruby with Ruby on Rails or JavaScript with Node/Express. Both paths will guide you in building skills, building up a portfolio of projects, and becoming a great developer. There is no right answer here on what is the next best path. Both paths will help you become a great developer. The important thing is to not switch halfway though, this will only set you back.
 
-Learning how to program is **much** more than just learning a language. It's about learning how to research and solve difficult problems, building an engineering mindset, working on complex problems and codebases, communicating effectively with other developers, and **much** more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
+Learning how to program is **much** more than just learning a language. It's about learning how to break down your problems into achievable steps, research and solve difficult problems, communicating effectively with other developers, and much more. Luckily, all these skills are transferrable to other languages and domains of software and because most languages are similar in concept and structure, moving into a different language will be much easier than learning your first.
```

Comment:
I like this, however, it should be an amalgamation of what was previously there and your new edits. We're presenting these on a high level so the concept of having an "engineering mindset" is something that the learner should be made aware of before they move forward.

Reply:
I intentionally removed that bit. Better to be explicit than sell some snake oil

Reply:
I don't think an "engineering mindset" is snakeoil

Reply:
@ryanford-dev But I know you have some very strong opinions on the term "engineer" so I can concede if you feel that it's really that false.